cmake/configs/nuttx_px4fmu-v2_default.cmake

```diff
@@ -1,5 +1,13 @@
 px4_nuttx_configure(HWCLASS m4 CONFIG nsh ROMFS y ROMFSROOT px4fmu_common IO px4io-v2)
 
+# user-configurable UART ports
+set(board_serial_ports
+	GPS1:/dev/ttyS3
+	TEL1:/dev/ttyS1
+	TEL2:/dev/ttyS2)
+
+set(px4_constrained_flash_build 1)
```

Comment:
We might want to look at cmake options for this kind of thing.
https://cmake.org/cmake/help/v3.0/command/option.html